worlds/psychonauts/docs/setup_en.md

```diff
@@ -0,0 +1,48 @@
+# Psychonauts Archipelago Setup Guide
+
+<h2 style="text-transform:none";>Quick Links</h2>
+
+- [Game Info Page](../../../../games/Psychonauts/info/en)
+- [Player Options Page](../../../../games/Psychonauts/player-options)
+- [PsychoRando Setup Guide](https://docs.google.com/document/d/1b7QOnOLmTSvdC7A1YK3bsSmhtSOsAMs0XF5j-tyE6Zw/edit?usp=sharing)
```

Comment:
The setup guide is fairly long, with a massive list of changes and full instructions for Astralathe and PsychoRando installation. Theoretically it could all be added to this file instead (or in addition to the google doc), but it would extend the length significantly. If length isn't a problem, I could add all the other steps in.

Reply:
The Google doc is for sure easier to change and won't require another pull request to update the instructions, for better or worse. 

From what I've seen in other games, it's pretty to have all the setup instruction here, even for games where they are pretty long. The list of change however is usually in the Game Page (`en_Psychonauts.md`). I don't think length should be a concern.